v6.txt

```diff
@@ -53,29 +54,46 @@ Example response: {"results":[{"index":0,"document":{"text":"Document to rank 1"
-- **X-With-Links-Summary** (optional): `true` to gather all links at the end of the response
-- **X-With-Images-Summary** (optional): `true` to gather all images at the end of the response
+- **X-With-Links-Summary** (optional): `all` to gather all links or `true` to gather unique links at the end of the response
+- **X-With-Images-Summary** (optional): `all` to gather all images or `true` to gather unique images at the end of the response
 - **X-With-Generated-Alt** (optional): `true` to add alt text to images lacking captions
 - **X-No-Cache** (optional): `true` to bypass cache for fresh retrieval
 - **X-With-Iframe** (optional): `true` to include iframe content in the response
 - **X-Return-Format** (optional): `markdown`, `html`, `text`, `screenshot`, or `pageshot` (for URL of full-page screenshot)
 - **X-Token-Budget** (optional): Specifies maximum number of tokens to use for the request
 - **X-Retain-Images** (optional): Use `none` to remove all images from the response
-
-Request body schema: {"application/json":{"url":{"type":"string","required":true},"options":{"type":"string","default":"Default","options":["Default","Markdown","HTML","Text","Screenshot","Pageshot"]}}}
+- **X-Respond-With** (optional): Use `readerlm-v2` for HTML to Markdown conversion, to deliver high-quality results for websites with complex structures and contents
+- **X-Set-Cookie** (optional): Forwards your custom cookie settings when accessing the URL, which is useful for pages requiring extra authentication. Note that requests with cookies will not be cached
+- **X-Proxy-Url** (optoinal): Utilizes your proxy to access URLs, which is helpful for pages accessible only through specific proxies
+- **X-Proxy** (optional): Sets country code for location-based proxy server. Use 'auto' for optimal selection or 'none' to disable
+- **DNT** (optional): Use `1` to not cache and track the requested URL on our server
+- **X-No-Gfm** (optional): Opt in/out features from GFM (Github Flavored Markdown). By default, GFM (Github Flavored Markdown) features is enabled. Use `true` to disable GFM (Github Flavored Markdown) features. Use `table` to Opt out GFM Table but keep the table HTML elements in response
+- **X-Locale** (optional): Controls the browser locale to render the page. Lots of websites serve different content based on the locale.
+- **X-Robots-Txt** (optional): Defines bot User-Agent to check against robots.txt before fetching content. Websites may allow different behaviors based on the User-Agent.
+- **X-With-Shadow-Dom** (optional): Use `true` to extract content from all Shadow DOM roots in the document.
+- **X-Base** (optoinal): Use `final` to follow the full redirect chain.
+- **X-Md-Heading-Style** (optinal): When to use '#' or '===' to create markdown headings. Set `atx` to use any number of "==" or "--" characters on the line below the text to create headings.
```

Comment:
`optional`. Just to be safe, check all instances of this. I see various typos :sweat: